bec_alerts/models.py

```diff
@@ -0,0 +1,198 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from django.contrib.postgres.fields import JSONField
+from django.db import connection, models
+from django.utils import timezone
+
+
+class User(models.Model):
```

Comment:
Given that Django has a `User` model, I probably would have named this something different. It's not a big deal for a prototype, but if you ever do a 2.0, I'd probably change this.